tests/unit/test_renderers.py

```diff
@@ -28,15 +31,18 @@ def test_renderers_get_renderer_workspace(
     path = tmp_path / ("test" + suffix)
     # use a csv as test data, it works for other types too
     path.write_text("a,b,c\n1,2,3")
-    content_cache_id = "65e73ba8-b"
+    content_cache_id = renderers.filesystem_key(path.stat())
 
-    time = 1709652904  # date this test was written
+    # Set the update time on the content file to the current time to ensure
+    # it's the time used for the last_modified date (otherwise in CI the
+    # template will be more recent because its just been cloned)
+    time = datetime.now().timestamp()
```

Comment:
What do you think about avoiding any dependency on the current time, and explicitly setting the times on the files we care about?